iothub/device/src/InternalClient.cs

```diff
@@ -893,30 +856,27 @@ public async Task SetMethodDefaultHandlerAsync(MethodCallback methodHandler, obj
             }
             finally
             {
-                _methodsDictionarySemaphore.Release();
+                _methodsSemaphore.Release();
 
                 if (Logging.IsEnabled)
-                {
                     Logging.Exit(this, methodHandler, userContext, nameof(SetMethodDefaultHandlerAsync));
-                }
             }
         }
 
         /// <summary>
         /// Sets a new delegate for the named method. If a delegate is already associated with
         /// the named method, it will be replaced with the new delegate.
+        /// </summary>
         /// <param name="methodName">The name of the method to associate with the delegate.</param>
         /// <param name="methodHandler">The delegate to be used when a method with the given name is called by the cloud service.</param>
         /// <param name="userContext">generic parameter to be interpreted by the client code.</param>
-        /// </summary>
 
         [Obsolete("Please use SetMethodHandlerAsync.")]
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public void SetMethodHandler(string methodName, MethodCallback methodHandler, object userContext)
         {
             if (Logging.IsEnabled)
-            {
```

Comment:
are the {} brackets gone now?

Reply:
Yup. Tim made a similar comment above:
https://github.com/Azure/azure-iot-sdk-csharp/pull/1940#discussion_r628430869

Reply:
We agreed we'd be okay with implicit body for logging statements, to cut down on how many lines they take.

Reply:
I've put them back for now. We can talk about it in scrum parking lot on Monday.